zk/datadog_checks/zk/zk.py

```diff
@@ -395,8 +398,8 @@ def parse_mntr(self, buf):
 
                 metrics.append(ZKMetric(metric_name, metric_value, metric_type, tags))
 
-            except ValueError:
-                self.log.warning("Cannot format `mntr` value. key=%s, value=%s", key, value)
+            except ValueError as e:
+                self.log.warning("Cannot format `mntr` value. key=%s, value=%s", key, value, str(e))
```

Comment:
```suggestion
                self.log.warning("Cannot format `mntr` value from `%s`: %s", line, str(e))
```

reformat this line too to fix the misleading log line